src/redis_cmd.cc

```diff
@@ -4083,22 +4106,70 @@ class CommandFetchFile : public Commander {
   CommandFetchFile() : Commander("_fetch_file", 2, false) {}
 
   Status Parse(const std::vector<std::string> &args) override {
-    path_ = args[1];
+    paths_sts_ = args[1];
     return Status::OK();
   }
 
   Status Execute(Server *svr, Connection *conn, std::string *output) override {
-    uint64_t file_size = 0;
-    auto fd = Engine::Storage::BackupManager::OpenDataFile(svr->storage_, path_,
-                                                           &file_size);
-    if (fd < 0) return Status(Status::DBBackupFileErr);
-    conn->Reply(std::to_string(file_size) + CRLF);
-    conn->SendFile(fd);
+    std::vector<std::string> paths;
```

Comment:
seems `files` was more precious than `paths`, as well as in replication.  But `paths` also ok to me. : )